.circleci/scripts/det_deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+det deploy $@
+status=$?
+# if we tried to bring up a cluster and it failed, recover by trying to bring the cluster down
+if [ $status -ne 0 ] && [ "$2" == "up" ] && [ "${CLUSTER_ID}" != "" ]; then
+    echo "CI error: det deploy $1 up failed, cleaning up cluster ${CLUSTER_ID}" >&2
```

Comment:
```suggestion
    echo "CI error: det deploy $1 up failed, attempting to clean up cluster ${CLUSTER_ID}" >&2
```
...Since this may fail too :)